### ISSUE TITLE
1.9.1 ready for deployment?

### ISSUE THREAD

**Issue by dparis** [clojurewerkz/money#16]
I saw that 1.9.1 was added to the repo, but it doesn't appear to have been deployed to clojars. Oversight, or just not ready yet?

**Comment by gws** [comment]
@dparis Are you referring to the `ChangeLog.md` addition, or something else?

**Comment by dparis** [comment]
@gws Yeah, the changelog indicates version 1.9.1 has been created to include the fixes you made in this commit:  https://github.com/clojurewerkz/money/commit/86d29f30c2698535ded663e18701c1e7e993a2aa

However, the latest version on clojars is still 1.9.0:  https://clojars.org/clojurewerkz/money

Just wondering if I've got the latest version at 1.9.0 or if I'm missing something.

**Comment by gws** [comment]
@dparis Well, the `ChangeLog.md` you're viewing is on the `master` branch. When I made that change it was only small enough to warrant a patch version number bump, but only assuming the release were cut from `master` at that point. Other changes may be made that warrant a minor or major version bump *before the next release is cut* and the `ChangeLog.md` on `master` would be updated accordingly, and be correctly reflected in the tagged version.

Some projects use a **??** placeholder, maybe this one does too, but I'm not familiar enough with the process, and tried to copy what I saw in the existing `ChangeLog.md` file.
